src/app.js

```diff
@@ -19,23 +15,15 @@ app.get('/', (req: Request, res: Response) => {
 const dialog = dialogflow();
 
 // Register handlers for Dialogflow intents
-
-dialog.intent('retrieve artifact', (conv, params) => {
-  logger.info(`parameters: ${conv.parameters}`);
-  logger.info(`Artifact: ${params.Artifact}`);
-  if (params.Artifact === 'Mindmap') {
-    conv.ask('Here is your Mindmap');
-    conv.ask(new Image({
-      url: mindmapimage,
-      alt: 'Your Mindmap',
-    }));
-  } else {
-    conv.ask('Here is your Architecture Sketch');
-    conv.ask(new Image({
-      url: architectureimage,
-      alt: 'Your Architecture Sketch',
-    }));
-  }
+dialog.intent(['Get Artifacts', 'Get latest Artifact'], (conv, params) => {
+  conv.ask('Esra isn\'t up right now but have a Cat instead!');
```

Comment:
Since this is user interaction, I'd prefer not to use the name Esra here. Imagine the D-School using this, they wont have any idea of what's going on. Instead something like: 'I can't reach the backend server, talk to the developers for now have a cat instead' ?